resources/testdriver-actions.js

```diff
@@ -441,12 +479,15 @@
       return data;
     },
 
-    pointerDown: function(actions, button) {
+    pointerDown: function(actions, button, width, height, pressure, tangentialPressure,
+                          tiltX, tiltY, twist, altitudeAngle, azimuthAngle) {
       let tick = actions.tickIdx;
       if (this.actions.has(tick)) {
         tick = actions.addTick().tickIdx;
       }
       this.actions.set(tick, {type: "pointerDown", button});
+      checkPointerProperties(this.actions.get(tick), width, height, pressure, tangentialPressure,
```

Comment:
I think this would be cleaer like
```
let actionProperties = setPointerProperties({type: "pointerDown", button}, width, height, ...);
this.actions.set(tick, actionProperties);

Reply:
Done.